### DISCUSSION TITLE
Is it possible to use Herbie to get HAFS data?

### DISCUSSION THREAD

**Discussion by ddxv** [blaylockbk/Herbie#225] [Ideas]
Does Herbie download HAFS data? I searched the documentation but didn't see that model mentioned yet.

**Comment by blaylockbk** [comment]
It does not yet. Looks like the HAFS data is on NOMADS, in grib format, and has an idx file. This should be straightforward to add a new template for it.

**Comment by ddxv** [comment]
Hello @blaylockbk, I saw the PR you already opened #227 for this, thanks so much for the quick help and I'm looking forward to getting my first try using Herbie! Let me know if you would like some help with documentation or anything, just know I am not familiar yet with weather models or Herbie.

**Reply by blaylockbk** [reply]
I merged #227 into main. I'll release a new version of Herbie as soon as the next MetPy release comes out (some other Herbie changes requires that update from MetPy).

You can see a quick HAFS demo here.
https://herbie.readthedocs.io/en/latest/user_guide/_model_notebooks/hafs.html

Play around with it and see how it works for you. Since Herbie is my hobby project, I can't promise to get to any changes as quickly as I did this (I did this while waiting at the car dealership for them to fix a recall part), so pull requests are welcome!

Thanks for the suggestion, this was pretty easy to implement and adds a new capability to Herbie, maybe it will even expanding its user base.

**Reply by ddxv** [reply]
Thanks, I was able to run the code in the notebook as you shared. Looks like it's working perfect, but I'll let you know if I encounter any issues. Just curious, is the summary data available that is in the `.hfsa.grib.stats.short` files on the server?

**Reply by blaylockbk** [reply]
It looks like those stats.short files are on the NOAMDAS server, but Herbie isn't doing anything with them right now.
